### ISSUE TITLE
[Livechat] - There is no route for the path: /livechat

### ISSUE THREAD

**Issue by leonardoterrao** [RocketChat/Rocket.Chat#4305]
I added support for multiple customers in livechat, but before the pull request I test and saw that the branch develop is not finding the path to '/livechat'.

To simulate the problem: Copy the installation script on livechat management screen. Add the script to another application. When trying to render the site, the console comes out the following message:

```
Uncaught ReferenceError: Autolinker is not defined ... livechat.jsi?_dc=3f6d9c296e10c7a0edcb09b6314e0488a4adb7e4:151
There is no route for the path: /livechat ... livechat.jsi?_dc=3f6d9c296e10c7a0edcb09b6314e0488a4adb7e4:100 There is no route for the path: /livechat/NANc6qEvD6zwf4xms
Uncaught TypeError: Cannot read property 'ServiceConfiguration' of undefined ... livechat.jsi?_dc=3f6d9c296e10c7a0edcb09b6314e0488a4adb7e4:125
```

I could not find the root cause. It happens to somebody else?
It will be some configuration that I forgot?

**Comment by deep0892** [comment]
Same happening with me too, definitely not a configuration problem.

**Comment by deep0892** [comment]
Has this been merged in 0.40.0 branch or just in the develop??

**Comment by deep0892** [comment]
there was a bug with livechat not opening , it was fixed with the above release, but i am facing some issues in livechat, the chat is not getting assigned to any user, i have an admin online and added to live chat, but i am not getting the messages sent from livechat to the admin.

Further, i am unable to create chat from incognito(chrome) , its prompts for usrname and emails, but keeps giving username already exists error.
I tried with multiple mails and username, it took the username first time but after that it keeps throwing error.
